src/lib/vrf_evaluator/vrf_evaluator.ml

```diff
@@ -13,7 +13,7 @@ module type CONTEXT = sig
 end
 
 (*Slot number within an epoch*)
-module Slot = Mina_numbers.Global_slot
+module Slot = Mina_numbers.Global_slot_since_genesis
```

Comment:
This is neither `Global_slot_since_genesis` or `Global_slot_since_hard_fork`. This is slot number within an epoch ( Consensus.Slot.t) and global slots are slot numbers across epochs. It is `Consensus.Slot.t` but that is not exposed and probably shouldn't be to avoid further confusion. Maybe this could be Mina_numbers.Nat.t?

Reply:
Made it `Nat.Make32 ()`